Gemfile

```diff
@@ -28,3 +28,6 @@ gem 'multi_json'
 # web & app server
 gem 'iodine'
 gem 'rack'
+
+# tests
+gem 'rspec'
```

Comment:
It might be useful to put it in a group, we are not going to use it in production, right? 
For Dockerfile we can use flag --without-group test to install everything except rspec

Reply:
yep, good point